person/views.py

```diff
@@ -1,3 +1,11 @@
+import json
 from django.shortcuts import render
+from .models import ASEMUser
 
-# Create your views here.
\ No newline at end of file
+def asem_user_list(request):
+    objects = ASEMUser.objects.all().values()
+    #objects_json = json.dumps(objects)
+    object_name = 'usuario'
+    title = "Gestion de Usuarios ASEM"
+    
```

Comment:
Codacy marks this empty line as a issue called trailing-whitespace, you can remove it to try to fix it.

Reply:
okey